draft-ietf-webpush-vapid.md

```diff
@@ -270,29 +285,21 @@ subscription to application servers that are able to provide proof of possession
 for the corresponding private key.
 
 This public key is then added to the request to create a push subscription as
-described in {{key}}.  The Crypto-Key header field includes exactly one public
-key.  For example:
+described in {{key}}.  The Crypto-Key header field includes exactly one ECDSA
+public key on the P-256 curve, encoded in the uncompressed form {{X9.62}} and
+base64url {{!RFC7515}}.  The example in {{ex-restrict}} shows a restriction to
+the key used in {{ex-push}}:
 
 ~~~
 POST /subscribe/ HTTP/1.1
 Host: push.example.net
-Crypto-Key: p256ecdsa=BBa22H8qaZ-iDMH9izb4qE72puwyvfjH2RxoQr5oiS4b
-                      KImoRwJm5xK9hLrbfIik20g31z8MpLFMCMr8y2cu6gY
+Crypto-Key: p256ecdsa=BA1Hxzyi1RUM1b5wjxsn7nGxAszw2u61m164i3MrAIxH
```

Comment:
can we use 'webpush' or 'vapid' instead of p256ecdsa ( to be more clear what the key is used for ) ?